WordPress/Classes/Utility/Analytics/WPAppAnalytics.m

```diff
@@ -118,6 +120,27 @@ - (void)applicationDidEnterBackground:(NSNotification*)notification
     [self trackApplicationClosed];
 }
 
+#pragma mark - Session
+
++ (NSInteger)sessionCount
+{
+    return [[NSUserDefaults standardUserDefaults] integerForKey:WPAppAnalyticsKeySessionCount];
+}
+
+- (NSInteger)incrementSessionCount
+{
+    NSInteger sessionCount = [[self class] sessionCount];
+    sessionCount++;
+
+    if (sessionCount == 1) {
+        [WPAnalytics track:WPAnalyticsStatAppInstalled];
+    }
+
+    [[NSUserDefaults standardUserDefaults] setInteger:sessionCount forKey:WPAppAnalyticsKeySessionCount];
```

Comment:
I would add a synchronize call here to catch the edge case that the app is force-quit before the change is synchronized automatically.